_docs/11-training.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: splash
+title: "Training"
+permalink: /docs/training/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Training page."
+last_modified_at: 2018-01-10T11:22:01-05:00
+---
+
+# Training Material
+## Below is a list of training available on Youtube.
+
+* [FPGA](https://www.youtube.com/watch?v=8cXOZ-V31kA&list=PL9B4edd-p2ajpeOI4k4Lr_fwXcRlwO3dv)
```

Comment:
Should these YouTube video links be grouped in some way? And the groups ordered? Maybe put the general and hardware related videos first (or last?), then the bare metal s/w related videos, and then the Linux s/w related videos?

Maybe in a later release consider having sub-headings to identify the groups, like for instance:
PolarFire SoC - Linux Applications Training
PolarFire SoC - Bare Metal Applications Training
etc.

Reply:
Yes, I think that is a good idea to group them as you have stated above. Worth doing this in this release?

Reply:
I think it is worth doing now - that is reshuffling the list of links. Leave the headings for another time.

Reply:
I have done the reshuffling of the list in the latest commit.